src/sentry/event_manager.py

```diff
@@ -2191,9 +2194,24 @@ def _save_aggregate_performance(jobs: Sequence[PerformanceJob], projects):
             ).select_related("group")
 
             new_grouphashes = set(group_hashes) - {hash.hash for hash in existing_grouphashes}
-            new_grouphashes_count = len(new_grouphashes)
 
             if new_grouphashes:
+                # temporary fix to limit group creation to grouphashes seen 3+ times in a 24-48 hour period
+                if settings.SENTRY_PERFORMANCE_ISSUES_REDUCE_NOISE:
+                    groups_to_create = new_grouphashes.copy()
+                    cluster_key = settings.SENTRY_PERFORMANCE_ISSUES_RATE_LIMITER_OPTIONS.get(
+                        "cluster", "default"
+                    )
+                    client = redis.redis_clusters.get(cluster_key)
+
+                    for new_grouphash in new_grouphashes:
+                        if should_create_group(client, new_grouphash) is not True:
```

Comment:
Nit: Since we return a bool from `should_create_group` the `is not True` isn't necessary